docs/kafka/kafka-bin-scripts-kafka/README.adoc

```diff
@@ -135,21 +135,24 @@ To enable the Kafka scripts to access a Kafka instance, you must configure a con
 
 . Create a file called `{property-file-name}`.
 
-. In the `{property-file-name}` file, set the SASL connection mechanism and the Kafka instance client credentials as shown in the following configuration. Replace the values with your own credential information.
+. In the `{property-file-name}` file, set the SASL connection mechanism and the Kafka instance client credentials as shown in the following configuration. Replace the values with your own credential information. {product-kafka}  supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use.
 +
 --
 .Setting server and credential values
 [source,subs="+quotes"]
 ----
-sasl.mechanism=PLAIN
+sasl.mechanism=OAUTHBEARER
 security.protocol=SASL_SSL
 
-sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \
-  username="__<client_id>__" \
-  password="__<client_secret>__" ;
+sasl.oauthbearer.token.endpoint.url=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
```

Comment:
There's an attribute we can use for the token URL. It's `{sso-token-url}`.

Reply:
Thanks - will change that.